DigitalLearningSolutions.Web/Styles/trackingSystem/allDelegates.scss

```diff
@@ -27,3 +28,8 @@
     width: 100%;
   }
 }
+
+.sort-dropdown {
+    margin-right: nhsuk-spacing(2);
+    margin-bottom: nhsuk-spacing(2);
+}

```

Comment:
So there is some space between the two dropdowns

Reply:
We shouldn't need this assuming we take the styling from the previous sort partial. And if we do need this (or similar), this isn't the correct place for it to go, it should be in the sort.scss file as it will need to be common to all pages with sort.

Reply:
I've made the partial, but the styling is different. Not 100% sure what to do about it because the layout is different.

Reply:
In that case, we can keep this styling (or a slightly modified version of it), but it should go in the sort.scss file. We'll need it for any page using your new partial.

Happy to have a chat about the structure of the scss/any issues with the styling if you want.

Reply:
Okay, so I've kind of commonised some of the styling.

Delegates Sort no-JS has same styling as Admins Filter w/JS.

Delegates Sort w/JS has both dropdowns side-by-side for large desktop, then one over the other for smaller screen sizes. The sort lines up with the search (same width).